.changeset/healthy-ties-beam.md

```diff
@@ -0,0 +1,5 @@
+---
+"react-magma-docs": patch
+---
+
+800 fix focus state
```

Comment:
This changes the Dropdown `closeDropdown` functionality, so I'd prefer this get tagged under `"react-magma-dom": major`. The description of the change should say that and maybe mention that modal examples were updated as well.